specs/toolbar.md

```diff
@@ -0,0 +1,127 @@
+# Toolbar
+
+## Overview
+As defined by the W3C:
+
+> A toolbar is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+> 
+> When a set of controls is visually presented as a group, the toolbar role can be used to communicate the presence and purpose of the grouping to screen reader users. Grouping controls into toolbars can also be an effective way of reducing the number of tab stops in the keyboard interface.
+
+Based on the above definition, it appears that as it currently stands, all elements within a toolbar _should_ be interactive controls. This includes support for [disabled controls](#disabled-controls).
+
+### Use Cases
+
+Used anywhere someone may want to visually and structurally group related interactive controls.
+
+- Tom is reading his favorite internet newspaper. Upon finishing an article, he notices that it has accrued nearly one thousand comments. Tom decides to read the comments and he ends up writing a response to someone he disagrees with. As he gets ready to submit, he realizes it will be easier for others to read his comment if he formats it. Tom sees a toolbar with formatting options and is able to bold all the words he deems important. He submits his comment and waits to see what happens next.
+
+### Prior Art/Examples
+- [Fluent UI](https://fluentsite.z22.web.core.windows.net/components/toolbar/definition)
+- [Material UI](https://material-ui.com/api/toolbar/)
+- [WAI-ARIA](https://w3c.github.io/aria-practices/examples/toolbar/toolbar.html)
+
+---
+
+### API
+
+*Component name:*
+- `fast-toolbar`
+
+*Attributes:*
```

Comment:
Do we need a prop to specify where initial focus goes if not the first item?

Reply:
I think the ideal scenario here would be to place focus only if focus has not already been placed within the toolbar. In that scenario, implementors can use existing methods (`.focus()`) on their HTMLElement, which seems more intuitive than a specialized API.